Sources/Datadog/RUM/RUMMonitor/Scopes/RUMScopeDependencies.swift

```diff
@@ -25,19 +25,11 @@ internal struct RUMScopeDependencies {
     let appStateListener: AppStateListening
     let deviceInfo: RUMDevice
     let osInfo: RUMOperatingSystem
-    let userInfoProvider: RUMUserInfoProvider
     let launchTimeProvider: LaunchTimeProviderType
-    let connectivityInfoProvider: RUMConnectivityInfoProvider
-    let serviceName: String
-    let applicationVersion: String
-    let sdkVersion: String
-    let source: String
     let firstPartyURLsFilter: FirstPartyURLsFilter
     let eventBuilder: RUMEventBuilder
     let eventOutput: RUMEventOutput
     let rumUUIDGenerator: RUMUUIDGenerator
-    /// Adjusts RUM events time (device time) to server time.
-    let dateCorrector: DateCorrectorType
     /// Integration with Crash Reporting. It updates the crash context with RUM info.
     /// `nil` if Crash Reporting feature is not enabled.
     let crashContextIntegration: RUMWithCrashContextIntegration?
```

Comment:
Nice to see it shrink 💪. I think that ultimately we should only keep RUM-specific properties in there and everything else should be passed through `SDKContext`, WDYT @maxep? Some of it is even possible now - e.g. I can see `sdkInitDate` and `MobileDevice` (transitive input for `RUMDevice` and `RUMOperatingSystem`) already defined in `DatadogV1Context`. If we agree, I think it's best to tackle them in separate PR.

Reply:
Ah indeed, that's a good point! Also makes me realise that I forgot to remove the `eventBuilder` and `eventOutput`.
I've created a dedicated story for managing the context in a single queue, [RUMM-2299](https://datadoghq.atlassian.net/browse/RUMM-2299), I can add that in the requirements!